scripts/install_rstudio.sh

```diff
@@ -58,6 +51,11 @@ if [ "$UBUNTU_CODENAME" = "focal" ]; then
     UBUNTU_CODENAME="bionic"
 fi
 
+# TODO: remove this workaround for Ubuntu 24.04
+if [ "$UBUNTU_CODENAME" = "noble" ]; then
+    UBUNTU_CODENAME="jammy"
+fi
```

Comment:
I believe this will need to be removed after a new version of RStudio is released.